app/app.google-services.json.debug

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
What happens if you remove this file?

Reply:
IIRC there were issues with the build failing on CI. Let's find out.